compile.sh

```diff
@@ -5,3 +5,5 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+dotnet publish csharp/csharp.csproj --ucr --nologo -c Release -o csharp-aot /p:PublishAot=true /p:PublishTrimmed=true
+dotnet publish csharp/csharp.csproj --ucr --nologo -c Release -o csharp
```

Comment:
`publish` now defaults to `Release` automatically so there is no more need to specify it. Same applies to `--use-current-runtime`. Please note that `PublishAot` implies trimming (well, a modified version of it done by ILC). You cannot set it to false, nor need to specify it. both of these can be just `dotnet publish csharp/csharp.csproj -o csharp-aot /p:PublishAot=true` and `dotnet publish csharp/csharp.csproj` respectively.

Reply:
Wasn't sure about the defaults, so I included the arguments just to be sure, thanks for pointing out they're redundant, removed them